TestingDangerApp/ViewController.swift

```diff
@@ -1,15 +1,16 @@
 import UIKit
 
 class ViewController: UIViewController {
-    lazy var label: UILabel = {
+    
+    lazy var label:UILabel = {
         let label = UILabel()
         label.text = "Olá mundo!"
         label.font = UIFont.systemFont(ofSize: 24)
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
     
-    override func viewDidLoad() {
+    override func viewDidLoad(){
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Edu15/TestingDangerApp/blob/45c1387bfb9d68560161334ef339af285201ad18/TestingDangerApp/ViewController.swift#L13"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>ViewController.swift:13</code></td>
    </tr>
  </tbody>
</table>